src/main/java/com/couchbase/lite/store/SQLiteViewStore.java

```diff
@@ -358,45 +434,68 @@ public void emit(Object key, Object value) {
                     keepGoing = cursor.moveToNext();
                     continue;
                 }
-                String revId = cursor.getString(3);
-
-                boolean noAttachments = cursor.getInt(4) > 0;
+                String revID = cursor.getString(3);
                 boolean deleted = cursor.getInt(5) > 0;
                 String docType = checkDocTypes ? cursor.getString(6) : null;
 
+                // Skip rows with the same doc_id -- these are losing conflicts.
+                // NOTE: Or Skip rows if 1st column is null
+                // https://github.com/couchbase/couchbase-lite-java-core/issues/497
+                ArrayList<String> conflicts = null;
+                boolean isNull;
                 while ((keepGoing = cursor.moveToNext()) &&
-                        (cursor.isNull(0) || cursor.getLong(0) == docID)) {
-                    // Skip rows with the same doc_id -- these are losing conflicts.
-                    // NOTE: Or Skip rows if 1st column is null
-                    // https://github.com/couchbase/couchbase-lite-java-core/issues/497
+                        ((isNull = cursor.isNull(0)) || cursor.getLong(0) == docID)) {
+                    if (isNull)
+                        continue;
+                    if (!deleted) {
+                        if (conflicts == null)
+                            conflicts = new ArrayList<String>();
+                        conflicts.add(cursor.getString(3));
+                    }
                 }
 
+                long realSequence = sequence; // because sequence may be changed, below
                 if (minLastSequence > 0) {
                     // Find conflicts with documents from previous indexings.
-                    String[] selectArgs2 = {Long.toString(docID), Long.toString(minLastSequence)};
-
                     Cursor cursor2 = null;
                     try {
+                        String[] selectArgs2 = {Long.toString(docID), Long.toString(minLastSequence)};
                         cursor2 = store.getStorageEngine().rawQuery(
                                 "SELECT revid, sequence FROM revs "
                                         + "WHERE doc_id=? AND sequence<=? AND current!=0 AND deleted=0 "
-                                        + "ORDER BY revID DESC "
-                                        + "LIMIT 1", selectArgs2);
```

Comment:
This SQL statement is for getting all conflicting revisions to construct `_conflicts` for line 532.